python/torch_mlir/extras/fx_importer.py

```diff
@@ -208,28 +208,33 @@
 }
 
 
-def sparsity_encoding(shape: torch.Size, sparse_layout: torch.layout) -> str:
+def sparsity_encoding(
+    shape: torch.Size, sparsity: Tuple[torch.layout, int, int]
```

Comment:
Use lowercase `tuple` type descriptors in new code. These capitalized versions are deprecated

Reply:
Ah, thanks for pointing that out.